### PR TITLE
add lvm storage operator

### PR DESCRIPTION
**Pull request by borball**
add lvm storage operator

### CODE REVIEW

**Comment by adetalhouet** [comment]
@borball Thank you for the contribution.
Please fix your PR based on the various failing checks. 
Thanks

**Comment by borball** [comment]
@adetalhouet Thanks for the comments, can you please review the changes again?

File: lvm-storage/operator/base/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: OpenShift LVM Storage Operator
+  labels:
+    openshift.io/cluster-monitoring: 'true'
+  name: openshift-storage
```
Comment (adetalhouet) [inline lvm-storage/operator/base/namespace.yaml:8]: don't you want to target a different namespace?

File: lvm-storage/operator/overlays/kustomization.yaml
Comment (adetalhouet) [inline lvm-storage/operator/overlays/kustomization.yaml:1]: this should probably be under stable-4.13 folder, and not under overlays

**Review by adetalhouet — COMMENTED** [review]
please review all the comments and address them, including the one from `lint-yaml` GitHub action.

File: lvm-storage/operator/base/namespace.yaml (hunk 1 above)
Comment (borball) [reply lvm-storage/operator/base/namespace.yaml:8]: I just the deault ns in the package manifest and doc reference so to align:

```
$ tasty info lvms-operator
Providing information on operator lvms-operator
source:  redhat-operators
channels:  [stable-4.12]
defaultchannel:  stable-4.12
target namespace:  openshift-storage
csv:  lvms-operator.v4.12.1
description:  Logical volume manager storage provides dynamically provisioned local storage.

```

File: lvm-storage/operator/overlays/kustomization.yaml
Comment (borball) [reply lvm-storage/operator/overlays/kustomization.yaml:1]: I see my new commits now. any way to retrigger the workflow to do a re-check? @adetalhouet

**Review by adetalhouet — APPROVED** [review]

**Comment by adetalhouet** [comment]
Thanks @borball - all good now.

**Comment by borball** [comment]
Thank you! @adetalhouet

**Pull request merged**